source/tutorials/nixos/nixos-configuration-on-vm.md

```diff
@@ -180,13 +180,19 @@ ls -R ./result
     run-nixos-vm
 ```
 
-Run the virtual machine:
+To run the virtual machine in a new window (requires GUI environment):
 
 ```shell-session
 ./result/bin/run-nixos-vm
 ```
 
-This command opens a window that shows the boot process of the virtual machine and ends at the `gdm` login screen where you can log in as `alice` with the password `testpw`.
+To run it in the current terminal instead, pass `-nographic`:
+
+```shell-session
+./result/bin/run-nixos-vm -nographic
+```
+
+You will see the boot process of the virtual machine and end at the `gdm` login screen where you can log in as `alice` with the password `testpw`.
```

Comment:
I strongly recommend against describing perceptions, as not everyone can even have them. Describe what the machine does instead.

```suggestion
This will show the boot process of the virtual machine and end at the `gdm` login screen where you can log in as `alice` with the password `testpw`.
```

I currently can't check it myself: will that indeed be a terminal prompt? Isn't GDM a graphical tool?